commits_panel.go

```diff
@@ -105,6 +106,25 @@ func handleCommitSquashDown(g *gocui.Gui, v *gocui.View) error {
 	return handleCommitSelect(g, v)
 }
 
+func handleCommitFixup(g *gocui.Gui, v *gocui.View) error {
+	if len(state.Commits) == 1 {
+		return createErrorPanel(g, "You have no commits to squash with")
+	}
+	branch := getSelectedBranch(v)
```

Comment:
to get the current branch you can just use `state.Branches[0]` because it's always the branch at the top of the Branches panel. Because it's attached to state, you don't need to send it in as an argument to gitRebaseTwoCommits, you can just get it there (state is a global variable)